Demo/Fullscreen/AddressView/AddressViewDemoView.swift

```diff
@@ -54,6 +58,33 @@ public class AddressViewDemoView: UIView, Tweakable {
             self.addressView.centerMap(location: location, regionDistance: 1200, animated: false)
         })
 
+        let coordinatesFor0010 = [[10.7244, 59.9181], [10.7252, 59.9185], [10.7264, 59.9192], [10.727099999999998, 59.919799999999995], [10.7274, 59.92], [10.727899999999998, 59.9202], [10.728499999999999, 59.9199], [10.7294, 59.919599999999996], [10.730299999999998, 59.9192], [10.729899999999999, 59.9184], [10.731199999999998, 59.917399999999994], [10.732099999999999, 59.9173], [10.7326, 59.917199999999994], [10.734699999999998, 59.9165], [10.734299999999998, 59.9161], [10.733899999999998, 59.9159], [10.7334, 59.9156], [10.7327, 59.9154], [10.730899999999998, 59.9153], [10.730199999999998, 59.9153], [10.7294, 59.9154], [10.728399999999999, 59.9154], [10.7266, 59.9154], [10.7252, 59.9155], [10.723999999999998, 59.9154], [10.723299999999998, 59.9154], [10.7228, 59.9154], [10.7227, 59.9157], [10.7228, 59.9161], [10.7229, 59.9163], [10.722999999999999, 59.916799999999995], [10.722999999999999, 59.916999999999994], [10.7234, 59.9173], [10.7244, 59.9181]]
+        let polygonDemoData = [
+            ["0010", "OSLO", [coordinatesFor0010]],
+            ["Multi polygons", "OSLO", [coordinatesFor0010, [[10.74, 59.92], [10.75, 59.92], [10.74, 59.91]]]]
+        ]
+        polygonDemoData.forEach({
+            let postalCode = ($0[0] as? String) ?? ""
+            let postalAreaName = ($0[1] as? String) ?? ""
+            let areas = ($0[2] as? [[[Double]]]) ?? [[[Double]]]()
```

Comment:
Is it possible to make it more readable? 😅

Reply:
What is difficult about that line 😜

Reply:
I made an update that I think makes this a bit better 😄